fe/fe-core/src/main/java/com/starrocks/load/routineload/KafkaTaskInfo.java

```diff
@@ -156,9 +154,9 @@ public TRoutineLoadTask createRoutineLoadTask() throws UserException {
             throw new MetaNotFoundException("table " + routineLoadJob.getTableId() + " does not exist");
         }
         tRoutineLoadTask.setTbl(tbl.getName());
-        // label = job_name+job_id+task_id
-        String label =
-                Joiner.on("-").join(routineLoadJob.getName(), routineLoadJob.getId(), DebugUtil.printId(id));
+        //  label = job_name+job_id+task_id
```

Comment:
you can remove this code if already define label

Reply:
done